File: S2B2/Lista-3/python/Programa Exercicio_1.py
cand_1 = 0;
cand_2 = 0;
cand_3 = 0;
cand_4 = 0;
branco = 0;
nulo   = 0;
total  = 0;
cont   = 's';

while (cont == 's' or cont == 'sim'):
	total = total + 1;
	print("Digite o numero do seu candidato:\n");
	print("1 - Tiririca\n");
	print("2 - Batoré\n");
	print("3 - Vovó da Fiel\n");
	print("4 - Mulher Pêra\n");
	print("5 - Branco\n");
	print("\n");
	num_voto = input();
	num_voto = int(num_voto);

	if (num_voto == 1):
		cand_1 = cand_1 + 1;
	elif (num_voto == 2):
		cand_2 = cand_2 + 1;
	elif (num_voto == 3):
		cand_3 = cand_3 + 1;
	elif (num_voto == 4):
		cand_4 = cand_4 + 1;
	elif (num_voto == 5):
		branco = branco + 1;
	else:
		nulo = nulo + 1;

	print("Mais algum eleitor para votar? [s/n]");
	cont = input();

print("Apuração dos votos");
print("Votos para o Tiririca: " + str(cand_1) + "\n");
print("Votos para o Batoré: " + str(cand_2) + "\n");
print("Votos para o Vovó da Fiel: " + str(cand_3) + "\n");
print("Votos para o Mulher Pêra: " + str(cand_4) + "\n");
print("\n");
print("Votos nulos:" + str(nulo) + "\n");
print("Votos em branco:" + str(branco) + "\n");
print("\n");
print("A percentagem de votos nulos sobre o total de votos: " + str(nulo * total / 100));
print("A percentagem de votos em branco sobre o total de votos: " + str(branco * total / 100));

File: S2B2/Lista-3/c/Programa exercicio_1.c
#include <stdio.h>

main ()
{
	int cand_1, cand_2, cand_3, cand_4, branco, nulo, num_voto, total;
	char cont[1];

	cand_1 = 0;
	cand_2 = 0;
	cand_3 = 0;
	cand_4 = 0;
	branco = 0;
	nulo   = 0;
	total  = 0;
	cont   = 's';

	while (strcomp(cons,'s'))
	{
		total++;
		printf("Digite o número de seu candidato:\n");
		printf("1-Tiririca\n");
		printf("2-Batoré\n");
		printf("3-Vovó da Fiel\n");
		printf("4-Mulher Pêra\n");
		printf("5-Branco\n");
		printf("\n");
		scanf("%i",&num_voto);

		switch (num_voto)
		{
			case 1:
				cand_1 ++;
				break;

			case 2:
				cand_2 ++;
				break;

			case 3:
				cand_3 ++;
				break;

			case 4:
				cand_4 ++;
				break;

			case 5:
				branco ++;
				break;

			else:
				nulo++;
		}

		printf("Mais algum eleitor para votar?\n[s/n]");
		scanf("%s",&cont);
	}

	printf("Apuração dos votos\n");
	printf("Votos para Tiririca:", cand_1,"\n");
	printf("Votos para Batoré:",cand_2,"\n");
	printf("Votos para Vovó da Fiel:",cand_3,"\n");
	printf("Votos para Mulher Pêra:",cand_4,"\n");
	printf("\n");
	printf("Votos nulo:",nulo,"\n");
	printf("Votos em brancos:",branco,"\n");
	printf("\r");
	printf("A percentagem de votos nulos sobre o total de votos: ", nulo * total / 100);
	printf("A percentagem de votos em branco sobre o total de votos: ", branco * total / 100);

	system("Pause");
}